assets/js/src/core/components/tree/components/context-menu/asset-tree-context-menu.tsx

```diff
@@ -78,14 +224,27 @@ export const AssetTreeContextMenu = (props: AssetTreeContextMenuProps): React.JS
 
   return (
     <>
+      <InputModal
+        initialValues={ defaultValue }
+        label={ 'Please enter the new name' }
+        onSubmit={ onRenameSubmit }
+        title={ 'Rename' }
+      />
+
+      <ConfirmationModal
+        onSubmit={ onConfirmation }
+        text={ confirmationText }
+        title={ 'Confirmation' }
```

Comment:
Please add translation keys for those messages

Reply:
[same](https://github.com/pimcore/studio-ui-bundle/pull/564#discussion_r1794838938)